docker/register.sh

```diff
@@ -103,14 +104,18 @@ setup_operator() {
         exit 1
     fi
 
-    
     if [ "$DEPLOY_ENV" = "TESTNET" ]; then
-        # TODO: remove this and replace with a check the AVS key has a balance
-        cast s "$public_key" --value 50000000000000000 --private-key "$FUNDED_KEY" -r "$LOCAL_ETHEREUM_RPC_URL" > /dev/null 2>&1
+        balance=$(cast balance "$public_key" --rpc-url "$LOCAL_ETHEREUM_RPC_URL")
```

Comment:
Could we really rename public key to address? Cuz that is what it is (see line 92)